app/assets/javascripts/listings/components/browse-listings.html.slim

```diff
@@ -63,7 +63,7 @@ section.results-section#listing-results ng-if="!$ctrl.parent.openListings.length
           p.margin-bottom--half
             | {{ $ctrl.noMatchesLabel() }}
           p.t-small.c-steel.no-margin
-            | <a class="lined" ui-sref="dahlia.additional-resources">Click here</a> for other rental and ownership affordable housing opportunities.
+            | <a class="lined" ui-sref="dahlia.additional-resources">{{'label.click_here' | translate}}</a> {{'listings.for_other_lisitings' | translate}}
```

Comment:
This should be combined into 1 string with the html embedded if possible, to help with word order or phrasing differences. We have a couple of examples of including links in the locale-en.json file if you want to check it out. You can pass the ui-sref in as a variable if it makes sense.